chef/core/src/main/java/org/jclouds/karaf/chef/core/ChefHelper.java

```diff
@@ -50,6 +52,10 @@
     public static final String JCLOUDS_CHEF_VALIDATOR_KEY_FILE = "JCLOUDS_CHEF_VALIDATOR_KEY_FILE";
     public static final String JCLOUDS_CHEF_VALIDATOR_CREDENTIAL = "JCLOUDS_CHEF_VALIDATOR_CREDENTIAL";
     public static final String JCLOUDS_CHEF_ENDPOINT = "JCLOUDS_CHEF_ENDPOINT";
+    
+    public static final TypeToken<ApiContext<ChefApi>> CHEF_TOKEN = new TypeToken<ApiContext<ChefApi>>() {
+       private static final long serialVersionUID = 1L;
```

Comment:
In this case, I think there's no need to propagate the warnings everywhere. It looks cleaner to me this way. WDYT?